texttunnel/models.py

```diff
@@ -0,0 +1,19 @@
+from texttunnel import chat
+
+GPT_3_5_TURBO = chat.Model(
+    name="gpt-3.5-turbo",
+    context_size=4000,
+    input_token_price_per_1k=0.002,
+    output_token_price_per_1k=0.004,
```

Comment:
Unfortunately there's no API endpoint that returns this information. Would've been ideal, because the rate limits actually differ between user accounts too.

You're right, this information will change and we don't want to update the package every time. I'll just remove it, then users can set this information for themselves.